internal/topology/variables/clusterclass_variable_validation_test.go

```diff
@@ -1023,12 +1023,266 @@ func Test_ValidateClusterClassVariable(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "Valid integer schema with CEL expression",
```

Comment:
Let's please change the names to a "pass if", "fail if" pattern

Reply:
Done.